file-formats/chkv/examples/z_aff_example_chkv.chkv.json

```diff
@@ -63,8 +61,7 @@
             {
               "sign": "exclude",
               "option": "equals",
-              "low": "P",
-              "high": ""
+              "low": "P"
```

Comment:
@bulcu is it ok to drop the empty `high`?

Reply:
Good question